OxfordBodleian/Ullendorff/BDLeu1.xml

```diff
@@ -182,12 +522,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         
                     </physDesc>
                     
-                <!--    <history>
+                    <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <origDate notBefore="1650" to="1849" evidence="lettering"/>
```

Comment:
It is quite difficult to narrow this time frame without external evidence. It is definitely not an elegant gwǝlḥ handwriting. Some graphic features, like the very separated loops of መ and a certain speed of execution, seem to suggest the second half of the time frame.
Since psalters have been extensively studied by Steve Delamarter, I wonder whether he has found some solid correlation between the ratio height/width of the codex and the age of the book. For instance, I remember that square book were typical of (or at least more widespread in) a certain age.